src/hooks/useConnectWallet/useProvider.ts

```diff
@@ -0,0 +1,88 @@
+import type { Web3Provider } from '@ethersproject/providers'
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { initializeConnector, Web3ReactHooks } from '@web3-react/core'
+import { getPriorityConnector } from '@web3-react/core'
+import { MetaMask } from '@web3-react/metamask'
+import { Connector, Web3ReactStore } from '@web3-react/types'
+import { WalletConnect } from '@web3-react/walletconnect'
+import { Buffer } from 'buffer'
+import { FALLBACK_JSON_RPC_URL, getNetwork, Web3ContextType } from 'hooks/useActiveWeb3React'
+import { useCallback } from 'react'
+
+export type Web3Connector = [Connector, Web3ReactHooks]
```

Comment:
Change this to `type Web3Connection = { connector: Connector; hooks: Web3ReactHooks }`